admin/CMakeLists.txt

```diff
@@ -55,27 +30,13 @@ configure_file("version.h.template"
 # =============================================================================
 # Install commands
 # =============================================================================
-# Function to install a file whose name ends in .template
-function(install_template_file)
-    set(_template_file "${ARGV0}")
-    # Deal with absolute paths - extract just the file name
-    get_filename_component(_template_base_name "${_template_file}" NAME)
-    # Remove the .template extension from the file name
-    string(REPLACE ".template" "" _new_file_name "${_template_base_name}")
-    install(
-        FILES "${_template_file}"
-        DESTINATION "."
-        RENAME "${_new_file_name}"
-    )
-endfunction()
-
 set(TEMPLATE_FILES
     "horace_on.m.template"
     "worker_v2.m.template"
-    "worker_4tests.m.template"
```

Comment:
Is this no longer used?

Reply:
Yup, no use in having it in a release package. See https://github.com/pace-neutrons/Horace/pull/640/commits/038e70e0da95dd1d18525e07cfd5a93ca7825b73